requests/src/main/scala/weco/api/requests/services/RequestsService.scala

```diff
@@ -57,14 +57,13 @@ class RequestsService(
   ): Future[List[(SierraHold, RequestedItemWithWork)]] =
     for {
       holdsMap <- sierraService.getHolds(patronNumber)
-
       itemLookupResults <- itemLookup.bySourceIdentifier(holdsMap.keys.toSeq)
 
       itemsFound = itemLookupResults.zip(holdsMap.keys).flatMap {
         case (Right(item), _) => Some(item)
         case (Left(itemLookupError: ItemLookupError), srcId) =>
           error(s"Error looking up item $srcId.", itemLookupError.err)
-          None
+          throw itemLookupError.err
```

Comment:
Can we do away with the `Some` in the other arm?

Reply:
yes